sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -24,7 +24,13 @@ import { getRetryAttemptTimeoutInMs } from "./util/utils";
 export class ServiceBusClient {
   private _connectionContext: ConnectionContext;
   private _clientOptions: ServiceBusClientOptions;
-
+  /**
+   * Host name of the service-bus namespace for which the client has been created for.
```

Comment:
There is a better doc string for this in EH. Can we copy that over? :)

```suggestion
   * The fully qualified namespace of the Service Bus instance for which this client is created.
   * This is likely to be similar to <yournamespace>.servicebus.windows.net.
```